client/resource_group/controller/config.go

```diff
@@ -102,12 +102,14 @@ type Config struct {
 	WriteBaseCost  RequestUnit
 	WriteBytesCost RequestUnit
 	CPUMsCost      RequestUnit
+	// The CPU statistics need to distinguish between different environments.
+	IsSingleGroupByKeyspace bool
```

Comment:
This name is too specified. Could we name it something more general? For example, `EnableTiDBProcessCPUMetering`. This should be more clear to tell its use and purpose.

Reply:
The purpose of this switch is to distinguish whether enable single group by keyspace feature.
I changed it to a private field and only changed the option in the controller